Add documentation on how NGINX Agent works and Install instructions (#74)
nginxinc/nginx-amplify-doc: amplify-guide.md

### Installing the Agent

> In order to be able to use NGINX Amplify to monitor your infrastructure, you need to install NGINX Amplify Agent on each system you wish to monitor.

Reviewer: ```suggestion
To monitor your infrastructure with NGINX Amplify, install the NGINX Amplify Agent on every system you want to monitor.
```

Author: AFAIK the amplify-guide.md file is not being used by the docs.nginx.com
